Charts/grist/h.sh

```
@@ -0,0 +1,27 @@
+GRIST_DATA_DIR=/persist/docs
+GRIST_DEFAULT_EMAIL=kencoveit@kencove.com
+GRIST_DOCKER_GROUP=grist
+GRIST_DOCKER_USER=grist
+GRIST_FORWARD_AUTH_HEADER=X-Forwarded-User
+GRIST_FORWARD_AUTH_LOGOUT_PATH=/_oauth/logout
+GRIST_HIDE_UI_ELEMENTS=billing,sendToDrive
+GRIST_HOST=0.0.0.0
+GRIST_INST_DIR=/persist
+GRIST_OIDC_IDP_CLIENT_ID=qei6trmgkhjrpsbkk74s8277s
```

Shellcheck (0.10.0)

[warning] GRIST_OIDC_IDP_CLIENT_ID appears unused. Verify use (or export if used externally).

(SC2034)